src/com/projectkorra/projectkorra/command/HelpCommand.java

```diff
@@ -108,35 +121,36 @@ public void execute(final CommandSender sender, final List<String> args) {
 			final CoreAbility ability = CoreAbility.getAbility(arg);
 			final ChatColor color = ability.getElement().getColor();
-
-			if (ability instanceof AddonAbility) {
-				if (ability instanceof PassiveAbility) {
-					sender.sendMessage(color + (ChatColor.BOLD + ability.getName()) + ChatColor.WHITE + " (Addon Passive)");
-				} else {
-					sender.sendMessage(color + (ChatColor.BOLD + ability.getName()) + ChatColor.WHITE + " (Addon)");
-				}
-
-				sender.sendMessage(color + ability.getDescription());
-
-				if (!ability.getInstructions().isEmpty()) {
-					sender.sendMessage(ChatColor.WHITE + this.usage + ability.getInstructions());
-				}
-
-				final AddonAbility abil = (AddonAbility) CoreAbility.getAbility(arg);
-				sender.sendMessage(color + "- By: " + ChatColor.WHITE + abil.getAuthor());
-				sender.sendMessage(color + "- Version: " + ChatColor.WHITE + abil.getVersion());
-			} else {
-				if (ability instanceof PassiveAbility) {
-					sender.sendMessage(color + (ChatColor.BOLD + ability.getName()) + ChatColor.WHITE + " (Passive)");
-				} else {
-					sender.sendMessage(color + (ChatColor.BOLD + ability.getName()));
-				}
-
-				sender.sendMessage(color + ability.getDescription());
-
-				if (!ability.getInstructions().isEmpty()) {
-					sender.sendMessage(ChatColor.WHITE + this.usage + ability.getInstructions());
+			final boolean isAddonAbility = ability instanceof AddonAbility;
+			final boolean isPassiveAbility = ability instanceof PassiveAbility;
+			
+			
+			sender.sendMessage(color + (ChatColor.BOLD + ability.getName()) + ChatColor.WHITE + (isAddonAbility ? (isPassiveAbility ? "(Addon Passive)" : " (Addon)") : (isPassiveAbility ? "(Passive)" : "")));
+			sender.sendMessage(color + ability.getDescription());
+			
+			if (!ability.getInstructions().isEmpty()) {
+				sender.sendMessage(ChatColor.WHITE + this.usage + ability.getInstructions());
+			}
+			
+			if (!isPassiveAbility) {
+				final ComponentBuilder bindShortcut = new ComponentBuilder();
+				for (int i = 1; i <= 9; i++) {
+					if (!bindShortcut.getParts().isEmpty()) {
+						bindShortcut.appendLegacy(color(this.bindSeperator));
+					}
+					
+					bindShortcut.appendLegacy(this.color(this.slotFormat.replace("{slot}", String.valueOf(i)).replace("{element_color}", color.toString())));
+					bindShortcut.event(new HoverEvent(HoverEvent.Action.SHOW_TEXT, new Text(new ComponentBuilder().appendLegacy( ChatColor.WHITE + this.hoverBind.replace("{ability}", color + ability.getName() + ChatColor.WHITE).replace("{slot}", color + String.valueOf(i) + ChatColor.WHITE)).create())));
+					bindShortcut.event(new ClickEvent(ClickEvent.Action.RUN_COMMAND, "/bending bind " + ability.getName() + " " + i));
 				}
+				bindShortcut.appendLegacy(this.color(this.bindEnd));
+				sender.spigot().sendMessage(new ComponentBuilder().appendLegacy(this.color(this.bindStart)).append(bindShortcut.create()).create());
```

Comment:
Is there any test to see if the player has permission to bind this ability before sending them the bind stuff? I can only see !isPassiveAbility

Reply:
Also good point, forgot about checking 😅

Reply:
added see line 135 of HelpCommand.java